### DISCUSSION TITLE
New game

### DISCUSSION THREAD

**Discussion by AlexArGC** [masterking32/MasterHamsterKombatBot#103] [Ideas]
AlexanderWayne, [06.08.2024 14:48]
Config is on cloud: config.uca.cloud.unity3d.com

AlexanderWayne, [06.08.2024 14:48]
{
  "common": {
    "appid": "f261c90f-1059-4ad0-9f0b-5281c9417f89",
    "userid": "ebfa7a8718f005043bd5c97e58ca1283",
    "sessionid": 1820221269829291500,
    "platform": "AndroidPlayer",
    "platformid": 11,
    "sdk_ver": "u2022.3.28f1",
    "runMode": "human-hub",
    "sdk_rev": "6bae5ce6b222",
    "session_count": 1,
    "localprojectid": "fbd3f693099e91442ba4ea3774531dd9",
    "build_guid": "7efc15b56bbe445bb686bc1a6c424c84",
    "deviceid": "5419a20f54fee26f6378e2bbb5461fec",
    "identity_token": "ebfa7a8718f005043bd5c97e58ca1283",
    "t_since_start": 4706376
  }
}

**Comment by argoarsiks** [comment]
Hi! What's it? do I understand chitr correctly, so you can somehow generate keys for new games?

**Comment by masterking32** [comment]
https://github.com/masterking32/MasterHamsterKombatBot/commit/256b7607743716383e0e54930f10e57c77e13e30
